.github/workflows/ci.yaml

```diff
@@ -1,21 +1,22 @@
 name: CI
 
 on:
-  - push
-  - pull_request
+  pull_request:
+  push:
+    branches: [ main ]
 
 jobs:
   rubocop:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Ruby
-      uses: ruby/setup-ruby@v1
-      with:
-        ruby-version: '3.1'
-        bundler-cache: true
-    - name: Run RuboCop
-      run: bundle exec rubocop
+      - uses: actions/checkout@v2
```

Comment:
v3 can be used.

Reply:
There's now a v3? Good shout!
They should have a rubocop for github actions that tells you when there are new versions available 😅